uniflow/op/model/neuron_utils.py

```diff
@@ -71,7 +71,10 @@ def get_instance_type() -> str:
         """
         url = "http://169.254.169.254/latest/meta-data/instance-type"
         try:
-            response = requests.get(url, timeout=1)
+            response = requests.put("http://169.254.169.254/latest/api/token", headers={"X-aws-ec2-metadata-token-ttl-seconds": "21600"}, timeout=1)
+            token = response.text
+            headers = {"X-aws-ec2-metadata-token": token}
+            response = requests.get("http://169.254.169.254/latest/meta-data/instance-type", headers=headers, timeout=1)
```

Comment:
Please fix build error https://github.com/CambioML/uniflow-llm-text-data-cleaning-cluster/actions/runs/7874169179/job/21492282209?pr=171. Make sure you setup your dev environment properly. https://www.notion.so/goldpiggy/Python-Linter-and-formatter-Setup-30fb3b81f0904af889832e4c697c5ec9?pvs=4